pkg/model/volumes.go

```diff
@@ -139,14 +139,13 @@ func (dev *Dev) getSourceSubPath(path string) string {
 		dev.parentSyncFolder = "."
 	}
 	rel, err := filepath.Rel(dev.parentSyncFolder, filepath.ToSlash(sourceSubPath))
+	if err != nil {
+		oktetoLog.Debugf("error on getSourceSubPath of '%s': %s", path, err.Error())
+	}
 	if filepath.IsAbs(sourceSubPath) {
-		if err != nil || strings.HasPrefix(rel, "..") {
-			if err != nil {
-				oktetoLog.Debugf("error on getSourceSubPath of '%s': %s", path, err.Error())
-			}
+		if strings.HasPrefix(rel, "..") {
```

Comment:
My understanding seeing the ticket is that we should just remove line `149` but the change in this if is changing the current behavior. 

Prior this change, we were executing the logic within this if there was an error calculating the relative path in line `141`. Now, we will never enter here if there is an error, as we are checking the result of that operation. Not sure if we want that or not cc @jLopezbarb

Reply:
> My understanding seeing the ticket is that we should just remove line 149 but the change in this if is changing the current behavior.

I thought the same while reading the ticket. I think that removing just the extra line is the safest move

> Prior this change, we were executing the logic within this if there was an error calculating the relative path in line 141. Now, we will never enter here if there is an error, as we are checking the result of that operation. Not sure if we want that or not cc @jLopezbarb

 We need to ensure that this logic is only executed when there is an error, this code is called when we retry a connection, and if executed twice it will modify the path and will return an error later in the code

Reply:
I'm not sure we should remove the err check in the if. I mean, if `sourceSubPath` is absolute and there was an error, it was always entering in this if and it could end up executing the logic in the `else` branch.

Removing the `err != nil` in the if, that path cannot happen again